libs/app-init/ui/src/lib/usb-access/usb-access-granted-modal.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { FunctionComponent } from "react"
+import { defineMessages } from "react-intl"
+import { formatMessage } from "app-localize/utils"
+import { ButtonType, IconType, ModalLayer, ModalSize } from "app-theme/models"
+import { Button, Modal, Typography } from "app-theme/ui"
+
+const messages = defineMessages({
+  title: {
+    id: "general.usbAccess.grantedModal.title",
+  },
+  description: {
+    id: "general.usbAccess.grantedModal.description",
+  },
+  buttonText: {
+    id: "general.usbAccess.grantedModal.buttonText",
+  },
+})
+
+interface Props {
+  opened: boolean
+  onClose: VoidFunction
+  onAction: VoidFunction
+}
+
+export const UsbAccessGrantedModal: FunctionComponent<Props> = ({
+  opened,
+  onClose,
+  onAction,
+}) => {
+  return (
+    <Modal opened={opened} layer={ModalLayer.UsbAccess} size={ModalSize.Small}>
+      <Modal.TitleIcon type={IconType.CheckCircleBlack} />
+      <Modal.Title>{formatMessage(messages.title)}</Modal.Title>
+      <Modal.CloseButton onClick={onClose} />
+      <Modal.DenseContent>
+        <Typography.P1>{formatMessage(messages.description)}</Typography.P1>
+      </Modal.DenseContent>
```

Comment:
No need for `<DenseContent>` when it has only one children. It's just a wrapper that changes the default modal's spacing.


```suggestion
      <Typography.P1>{formatMessage(messages.description)}</Typography.P1>
```

Reply:
https://github.com/mudita/mudita-center/pull/2573/commits/2c0eff22e8358004eefdd8e1e0ea0edf11f6ddef